### PR TITLE
PRIPAD-5150 Pull latest version uploaded

### PR DESCRIPTION
**Pull request by sebskuse**
- Allow the latest version of a platform specific app to be displayed on the bundle landing page. i.e. com.test.app on an iOS device will show com.test.app/ios/1.2 if 1.2 is the most recent version.

### CODE REVIEW

File: server/php/includes/views/superview.html [hunk 1]
```diff
@@ -14,8 +14,11 @@
             <p class='lead visible-md visible-lg'>Visit this page directly from your iPad, iPhone, iPod touch or Android device and you will be able to install an app directly on your device.</p>
             <hr />
         <div>
+        
         {content}
-
+        
+        <hr />
+        &copy; 2014 Quark Inc
```
Comment (steveanthony) [inline server/php/includes/views/superview.html]: Is this the footer view that was mentioned earlier?

Comment (sebskuse) [inline server/php/includes/views/superview.html]: Yep

Comment (dhardiman) [inline server/php/includes/views/superview.html]: Can we not commit this in a public repo please. Let's stick it in a text file to be loaded by the installation.

**Comment by sebskuse** [comment]
Updated

**Pull request merged**